### PR TITLE
Track ui

### PR DESCRIPTION
**Pull request by lsivasothy**
simulator screen shot - iphone 7 plus - 2018-03-13 at 17 16 01

### CODE REVIEW

**Comment by jesshanta** [comment]
UI looks great!

File: ChemoTracker/components/SymptomTracking/Form.js [hunk 1]
```diff
@@ -3,16 +3,21 @@ import { View, Text, StyleSheet, Dimensions } from 'react-native';
...
   state = {
-    buttonbg1: 'transparent',
+    buttonbg1: '#FFF',
     buttonbg2: 'transparent',
     buttonbg3: 'transparent',
     buttonbg4: 'transparent',
+    buttontxt1: '#FB5B1B',
+    buttontxt2: '#FFF',
+    buttontxt3: '#FFF',
+    buttontxt4: '#FFF',
```
Comment (jesshanta) [inline ChemoTracker/components/SymptomTracking/Form.js]: I would suggest putting the colour names into the color file and reference {color.white}

File: ChemoTracker/components/SymptomTracking/Form.js [hunk 2]
```diff
@@ -23,95 +28,63 @@ class FormScreen extends Component {
...
           <View style={styles.gradeButtons}>
-            <Button style={[styles.button, {backgroundColor: this.state.buttonbg1}]} textStyle={styles.buttonText}
-              onPressIn={()=>{
-                this.setState({buttonbg2:'transparent',buttonbg3:'transparent',buttonbg4:'transparent',buttonbg1:'#fff', selectedgrade:0});
+            <Button style={[styles.button, { backgroundColor: this.state.buttonbg1 }]} textStyle={[styles.buttonText, { color: this.state.buttontxt1 }]}
+              onPressIn={() => {
+                this.setState({
+                  buttonbg1: '#FFF', buttonbg2: 'transparent', buttonbg3: 'transparent', buttonbg4: 'transparent',
+                  buttontxt1: '#FB5B1B', buttontxt2: '#FFF', buttontxt3: '#FFF', buttontxt4: '#FFF',
+                  selectedgrade: 0
```
Comment (jesshanta) [inline ChemoTracker/components/SymptomTracking/Form.js]: i'd fix the colours here. If they change, it's way easier just the change the colour code in colors.js

File: ChemoTracker/styles/SymptomTracking/intervention.js [hunk 3]
```diff
@@ -40,7 +40,8 @@ export const styles = StyleSheet.create({
     borderColor: '#FB5B1B',
     height: height(5),
     padding: height(2),
-    width: isIphoneX() || isIphonePlus() ? width(40) : width(35)
+    width: isIphoneX() || isIphonePlus() ? width(40) : width(35),
+    borderRadius: width(40) / 2,
```
Comment (jesshanta) [inline ChemoTracker/styles/SymptomTracking/intervention.js:44]: i should probs use this in my code too lol

File: ChemoTracker/styles/SymptomTracking/track.js [hunk 4]
```diff
@@ -1,10 +1,26 @@
...
+  const { height, width } = d;
+  return (
+    Platform.OS === 'ios' && (height === 736 || width === 736)
+  );
+}
+
 export const headerStyles = StyleSheet.create({
-  headerBackground:{
+  headerBackground: {
     backgroundColor: '#F4FAFC'
```
Comment (jesshanta) [inline ChemoTracker/styles/SymptomTracking/track.js]: same colour variable as before

**Review by jesshanta — CHANGES_REQUESTED** [review]

**Review by jpatricia — APPROVED** [review]

**Pull request merged**